redbot/core/commands/help.py

```diff
@@ -209,9 +210,22 @@ async def format_command_help(
         description = command.description or ""
 
         tagline = (help_settings.tagline) or self.get_default_tagline(ctx)
-        signature = _(
-            "`Syntax: {ctx.clean_prefix}{command.qualified_name} {command.signature}`"
-        ).format(ctx=ctx, command=command)
+        signature = (
+            f"`{_('Syntax')}: {ctx.clean_prefix}{command.qualified_name} {command.signature}`"
+        )
```

Comment:
Something went wrong with conflict resolution here - these lines weren't changed in this PR.